datalad/resources/procedures/cfg_yoda.py

```diff
@@ -15,6 +15,30 @@
     check_installed=True,
     purpose='YODA dataset setup')
 
+to_modify = [
+    ds.pathobj / 'code' / 'README.md',
+    ds.pathobj / 'code' / '.gitattributes',
+    ds.pathobj / 'README.md',
+    ds.pathobj / 'CHANGELOG.md',
+    ds.pathobj / '.gitattributes',
+]
+
+dirty = [
```

Comment:
here it probably doesn't matter either way, but IMHO better to use generators more to gain some efficiency.  E.g. making here it into a `()`, and below condition into `if any(dirty):`

Reply:
Hmm, but wouldn't the generator be consumed by `any()` and leave me no info to put in the exception?

Reply:
ah, right. sorry I have missed that it was used below!  Sorry for the noise, disregard original comment ;)